src/profile-logic/marker-timing.js

```diff
@@ -95,23 +95,10 @@ export function getMarkerTiming(
         markerTimingsByName.push(markerTimingsRow);
       }
 
-      let continueSearching = false;
-
-      // Search for a spot not already taken up by another marker of this type.
-      for (let j = 0; j < markerTimingsRow.length; j++) {
-        const otherStart = markerTimingsRow.start[j];
-        const otherEnd = markerTimingsRow.end[j];
-        if (otherStart > marker.start + marker.dur) {
-          break;
-        }
-        if (otherEnd > marker.start) {
-          continueSearching = true;
-          break;
-        }
-      }
-
-      if (!continueSearching) {
-        // An empty spot was found, fill the values in the table.
```

Comment:
TBH I always found the previous algorithm difficult to follow.
So I'm quite happy to see this disappear.